clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -157,13 +158,23 @@
         </xs:annotation>
       </xs:element>
     </xs:sequence>
-    <xs:attribute name="auto-create" type="xs:boolean" use="optional" default="false">
+    <xs:attribute name="auto-create" type="xs:boolean" use="optional">
       <xs:annotation>
-        <xs:documentation> xml:lang="en">
+        <xs:documentation xml:lang="en">
           True if server side components should be automatically created if they are absent.
         </xs:documentation>
       </xs:annotation>
     </xs:attribute>
+    <xs:attribute name="client-mode" type="tc:client-mode-spec" use="optional">
+      <xs:annotation>
+        <xs:documentation> xml:lang="en">
+          The client connection behavior, either:
+           * expecting - expect a matching server configuration (fail otherwise)
+           * auto-create - expect a matching server configuration or create one if none is present
+           * auto-create-on-reconnect - as 'auto-create' but also auto-create on reconnection
+        </xs:documentation>
```

Comment:
Does it means that this won't be possible to have the client auto-create on connect first time, but not auto-create on network failure for example ? I didn't see how we could have the behaviour where we fail if the cluster was gone ?

Reply:
You can have the behavior you want. That's just 'auto-create' as it stands.